src/UserService.Models.Db/DbUser.cs

```diff
@@ -20,8 +24,37 @@ public class DbUser
         [Required]
         public bool IsActive { get; set; }
         public bool IsAdmin { get; set; }
+        public DateTime? CreatedAt { get; set; }
```

Comment:
need to add default value via fluent api. DefaultValue == 01/01/2021

Reply:
done